components/chef-run/i18n/en.yml

```diff
@@ -32,7 +32,7 @@ cli:
   description: |
     Chef Run is a tool to execute ad-hoc tasks using Chef.
   creating_config: "Creating config file in %1."
-  default_config_location: "Location of config file. Defaults to %1"
+  default_config_location: "Location of config file. Default: %1"
```

Comment:
I found this wording issue while looking at the help text - brought it in line with other descriptions